.golangci.example.yml

```diff
@@ -109,6 +109,10 @@ linters-settings:
   funlen:
     lines: 60
     statements: 40
+  gci:
+    # put imports beginning with prefix after 3rd-party packages;
+    # only support one prefix
+    local-prefixes: github.com/org/project
```

Comment:
Is it intended? Probably singular form `local-prefix` can be better here, but it will be inconsistent with goimports.

Also it would be great to pick prefix automatically from goimports settings, but seems like it is out of scope.

Reply:
Yep, it is intended, since i want to support multi-prefixes in feature, so keep same with `goimports` here.

`gci` handles empty lines between import groups and re-split them into groups aby setting, `goimports` can not do that.

Reply:
> how about use goimports settings if no settings for gci?

I like this